.circleci/config.yml

```diff
@@ -110,6 +129,8 @@ exports: &exports
     # Apart from using a DEV_TAG we use a different image ID to avoid polluting the tag
     # history of the production tag
     echo "export IMG_MODIFIER=-ci" >> $BASH_ENV
+    # Kubectl version, we use the same for all the builds since typically it's backwards compatible
+    echo "export KUBECTL_VERSION=v1.12.0" >> $BASH_ENV
```

Comment:
Do you need to export KUBECTL_VERSION here in the shared context? It seems to be used only once.

Actually it might make sense to bring here the HELM_VERSION instead.

Reply:
I wanted to give visibility of the version just in case it needs to be reused but it's not going to be probably the case.

You are right, we can save a few lines if we set HELM_VERSION here.